R/setup_package.R

```diff
@@ -117,8 +122,14 @@ allowed:
   # add LICENSE.md
   if (length(list.files(path, "LICEN(S|C)E")) == 0) {
     file.copy(
-      system.file(
-        file.path("generic_template", "gplv3.md"), package = "checklist"
+      switch(
+        license,
+        "GPL-3" = system.file(
+          file.path("generic_template", "gplv3.md"), package = "checklist"
+        ),
+        "MIT" = system.file(
+          file.path("generic_template", "mit.md"), package = "checklist"
```

Comment:
see the comment on  `create_package()`

Reply:
Done in [9bc3779](https://github.com/inbo/checklist/pull/92/commits/9bc3779f77df8ff73c358d6252dc313457334568)